packages/vats/src/core/boot-oracles.js

```diff
@@ -0,0 +1,283 @@
+// @ts-check
+/** @file Boot script for demoing oracles. Builds upon boot-psm to be used in integration tests. */
```

Comment:
move to /test/ directory?

Reply:
moved to `inter-protocol` since it depends on its price feed proposal and under `scripts` since it's not used in any automated tests